### PR TITLE
ISS-226 # JUnit5 Jupiter Load Extension for Performance Testing

### PR DESCRIPTION
**Pull request by authorjapps**
Passed now - https://travis-ci.org/authorjapps/zerocode/builds/525296505
_It was the pom issue_

This runs Jupiter tests in parallel for generating Load/Stress on the target system.
Extension name: `ZeroCodeParallelExtension`

Examples and usage are in this PR-
https://github.com/authorjapps/performance-tests/pull/3/files

Please have a look at the below parallel load examples(in the above PR)-

JUnit5LoadTest.java
JUnit5LoadCommonLoadTest.java
JUnit5LoadDifferentLoadTest.java

### CODE REVIEW

File: junit5-testing/src/test/java/org/jsmart/zerocode/jupiter/extension/ParallelLoadExtensionTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import org.jsmart.zerocode.core.domain.TestMappings;
+import org.jsmart.zerocode.tests.jupiter.JUnit5Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ParallelLoadExtensionTest {
```
Comment (siddhagalaxy) [inline junit5-testing/src/test/java/org/jsmart/zerocode/jupiter/extension/ParallelLoadExtensionTest.java]: 👍 
Pls add more examples into the HelloWorld Load repo.

File: junit5-testing/src/test/java/org/jsmart/zerocode/tests/loadjupiter/JUnit5LoadTest.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import org.jsmart.zerocode.core.domain.LoadWith;
+import org.jsmart.zerocode.core.domain.TestMapping;
+import org.jsmart.zerocode.core.domain.TestMappings;
+import org.jsmart.zerocode.jupiter.extension.ParallelLoadExtension;
+import org.jsmart.zerocode.tests.jupiter.JUnit5Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith({ParallelLoadExtension.class})
+public class JUnit5LoadTest {
```
Comment (siddhagalaxy) [inline junit5-testing/src/test/java/org/jsmart/zerocode/tests/loadjupiter/JUnit5LoadTest.java]: More examples in HelloWorld repo will be helpful...

Comment (authorjapps) [reply junit5-testing/src/test/java/org/jsmart/zerocode/tests/loadjupiter/JUnit5LoadTest.java]: will do.
Already a PR with few handy examples to unblock the user are there!
PR link: https://github.com/authorjapps/performance-tests/pull/3/files

File: junit5-testing/src/test/java/org/jsmart/zerocode/jupiter/extension/ParallelLoadExtensionTest.java (hunk 1 above)
Comment (authorjapps) [reply junit5-testing/src/test/java/org/jsmart/zerocode/jupiter/extension/ParallelLoadExtensionTest.java]: will do.
Already a PR with few handy examples to unblock the user are there!
PR link: https://github.com/authorjapps/performance-tests/pull/3/files

**Pull request merged**